blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -59,197 +70,279 @@ blueprint:
       default: []
       selector:
         action:
-    # inputs for looping custom actions on long button press events until the corresponding release event is received
     button_long_loop:
       name: Long button press - loop until release
       description: Loop the action until the button is released.
       default: false
       selector:
         boolean:
-    # inputs for enabling double press events
+    button_long_max_loops:
+      name: Long Button press - Maximum loops
+      description: Maximum amount of repeats.
+      default: 500
+      selector:
+        number:
+          min: 0
+          max: 5000
+          mode: slider
+          step: 1
     button_double_press:
       name: Expose button double press event
-      description: Choose whether or not to expose the virtual double press event for the button. Turn this on if you are providing an action for the button double press event.
+      description: Choose whether or not to expose the virtual double press event
+        for the button. Turn this on if you are providing an action for the button
+        double press event.
+      default: false
+      selector:
+        boolean:
+    button_double_initial_press:
+      name: Run short press action at once.
+      description: Choose whether or not to run the short press action at once, without waiting for the 2nd press.
+        When a second press is received the double press action will be run at once and the short press action will stop if it's still running.
       default: false
       selector:
         boolean:
-    # helpers used to properly recognize the remote button events
+    #Only requiered since it seems impossible to get the automations attribute dynamicly via a blueprint. 
```

```suggestion
    # helpers used to properly recognize the remote button events
```

we can keep the old comment here, so it won't confuse users. also, this way we're keeping the same format for all blueprints